frontend/ui/size.lua

```diff
@@ -26,7 +26,19 @@ defaults, please take a second to consider:
 ]]
 
 local dbg = require("dbg")
-local Screen = require("device").screen
+local Device = require("device")
+local Screen = Device.screen
+
+-- This function calculates reasonably large smallest size for items based on screen properties:
+-- px as the screen_height or screen_width, and screen_dpi.
+-- It is then used to calculate how many such items can fit in a screen.
+-- 72 is used because there are 72 points per inch (in printing press).
+-- limit is points (pt), i.e., what is minimum number of points to make this item readable.
+-- Items might be number of lines in the list (pt = 20), of mosaic grid cover size (pt = 75)
+local function size_max_items_norm(px, dpi, limit)
+    local maxItemsNorm = math.floor(px / dpi * 72 / limit + 0.5)
+    return maxItemsNorm
```

Comment:
A nitpick but those should be the other way around according to the coding style.

As in

```
local function calcMaxItemsEtc() -- lowerCamelCase
  local max_items_norm = etc -- snake_case
```